### DISCUSSION TITLE
Add a toggle to show score before the mod multipliers are applied next to the regular one on the leaderboard and allow sorting by it for local leaderboads

### DISCUSSION THREAD

**Discussion by Newbie1st** [ppy/osu#31024] [Ideas] [answered]
Should look something like this
![Inkedosu_2024-12-09_11-21-54_LI1](https://github.com/user-attachments/assets/b262a156-099c-4843-a01a-2886893f0c5e)

**Comment by peppy** [comment, marked as answer]
This was one of my plans from the outside, although it would probably just be a toggle at the top of the leaderboard to change the main score display rather than add a second number (space limited).

**Comment by bdach** [comment]
I'm confused, why would anyone want this in general?

Also this is going to be *veeeeeeery* difficult to pull off correctly for local scores. Online is a different matter entirely because of https://github.com/ppy/osu-web/pull/11211 but local scores?

**Reply by normalid-awa** [reply]
if you want to check out what's the value of a play where you have RX or NF sort of. Because of the NF will decrease the score so that this toggle allows you to see the score without mod.

**Reply by bdach** [reply]
Yeah I dunno.

Maybe it's not gonna be *that* difficult since I forgot total scores without mods are already in realm... as long as those numbers are roughly correct (they may not be for stable scores, especially score v2). But at least the sorting thing still seems weird, who'd want a leaderboard with RX scores potentially above DT or something?

**Reply by Newbie1st** [reply]
I was under the impression that the game already stores the pre-mods score in the score data. If it isn't the case, then simply dividing the post-mods score by the total mod multiplier should suffice? 

I use score thresholds to determine when to increase the rate or go for an SS in mania, so pulling out a calculator every time I set a score with down-rate and/or constant speed is annoying. Plus, there is an OD-dependant marvelous window now (and maybe custom client-side only judgements someday?), so increasing OD through difficulty adjust to train accuracy is a viable option too.

**Reply by normalid-awa** [reply]
I don't know how to explain it, like, you want to play a map, but there's a diff spike where you'll potentially fail there, so you put NF on, and hit it but you pass that spike so the NF became useless but it still deducted scores, but you still want to checks the value of that play. In that way this things became useful then.

**Reply by bdach** [reply]
> If it isn't the case, then simply dividing the post-mods score by the total mod multiplier should suffice?

Doesn't work if the mod multipliers ever change / get rebalanced. But yes we kinda do store no-multiplier total score now.
